addons/l10n_in_pos/static/src/js/receipt.js

```diff
@@ -2,16 +2,19 @@ odoo.define('l10n_in_pos.receipt', function (require) {
 "use strict";
 
 var models = require('point_of_sale.models');
+const Registries = require('point_of_sale.Registries');
 
-models.load_fields('product.product', 'l10n_in_hsn_code');
+Registries.PosModelRegistry.extend(models.Orderline, (Orderline) => {
```

Comment:
@caburj This syntax is not very clear. Couldn't we have something that looks more like the ComponentRegistry syntax?

(Maybe after having merged the @trj-odoo PR)

Reply:
Will do after merging TRJ's.

Reply:
How about this?

https://github.com/odoo-dev/odoo/pull/1084
https://github.com/odoo-dev/enterprise/pull/227